### DISCUSSION TITLE
Close Dropdown via Close button

### DISCUSSION THREAD

**Discussion by bryanu** [themesberg/flowbite-react#1235] [Q&A]
Using the React version of the Dropdown component as a Dropdown to add sort and filter options on a table.  The use can select sort direction and select tags for the filter, but need to know how to close the Dropdown when the user select either APPLY or Cancel Buttons.

**Comment by nigellima** [comment]
Could you provide a reproducible codesandbox/stackblitz?

By default it should close just fine when you click on one of the options. Is it not closing?
